### ISSUE TITLE
ClassNotFound error for blaze burner with Create 0.5.1

### ISSUE THREAD

**Issue by jshipley** [oierbravo/melter-forge-mod#9]
This mod crashes the game when you try to place a melter with Create 0.5.1.

**Comment by nanonestor** [comment]
Yeah Crete mod did some internal file changes and the class file that's being looked for is renamed/moved/changed.  This will need to be fixed in the melter mod before it can be used in the same instance as Create.

Someone has already done a GitHub PR for it for 1.19.2.  Hopefully the change can get do e for 1.18.2 also - I'll see if I can do the same changes in my own PR
